src/libstd/thread/mod.rs

```diff
@@ -713,22 +713,27 @@ struct Inner {
 
 #[derive(Clone)]
 #[stable(feature = "rust1", since = "1.0.0")]
-/// A handle to a thread.
+/// A handle to a thread, its just an abstract reference and as such
```

Comment:
"just" is a poor word for docs; this becomes better just dropping it.

Also, this summary is quite long; I think keeping the initial summary is fine, and putting this after:

    /// A handle to a thread.
    ///
    /// It is an abstract...